daml-lf/archive/BUILD.bazel

```diff
@@ -100,10 +130,10 @@ da_scala_library(
     name = "daml_lf_archive_reader",
     srcs = glob(["src/main/scala/**/*.scala"]),
     scalacopts = lf_scalacopts,
-    tags = ["maven_coordinates=com.digitalasset:daml-lf-archive-reader:__VERSION__"],
+    tags = ["maven_coordinates=com.digitalasset:daml-lf-archive-scala:__VERSION__"],
```

Comment:
As for the other change, this is potentially breaking since users will have to update their references.

Reply:
done